scene/gui/split_container.cpp

```diff
@@ -473,6 +459,15 @@ int SplitContainer::get_split_offset() const {
 	return split_offset;
 }
 
+void SplitContainer::clamp_split_offset() {
```

Comment:
Same here, depending on where it's used, see if you can send already computed values of `_get_child(0)` and second one.

Reply:
This method is exposed for GDScript and changing it will break compatibility, it was useless before making the changes because i used to clamp the split_offset internally, but now the split_offset is free and we have another `middle_sep` internal value represents the offset and in 4.4, it was renamed to `computed_split_offset` which makes sense.